pkg/admission/provider.go

```diff
@@ -1,93 +1,132 @@
 // TODO: only supports plain text with no auth, add security later
-// TODO: use keystone to access Neutron API
+// TODO: check keystone expiration date 
 package admission
 
 import (
-	resty "gopkg.in/resty.v1"
+	"gopkg.in/goose.v2/identity"
+	"gopkg.in/goose.v2/http"
+	"fmt"
+	"errors"
+)
+
+const (
+	ProviderDefaultRegion             = "RegionOne"
+	ProviderDefaultNetworkServiceName = "network"
+	ProviderTenantName                = "admin"
+	ProviderVersion                   = "v2.0"
+	ProviderNetworks                  = "networks"
+	ProviderPorts                     = "ports"
+	ProviderTokens                    = "tokens"
+	PostExpectedReturnCode            = 201
+	DeleteExpectedReturnCode          = 204
 )
 
 type Network struct {
```

Comment:
If you introduce the dependency to goose, why not use the network type from goose?